crates/teloxide-core/src/types/message.rs

```diff
@@ -348,7 +365,7 @@ pub struct MediaAnimation {
 
     /// For messages with a caption, special entities like usernames, URLs,
     /// bot commands, etc. that appear in the caption.
-    #[serde(default = "Vec::new")]
+    #[serde(skip_serializing_if = "Vec::is_empty", default = "Vec::new")]
```

Comment:
Let's first have `default` and _then_ `skip_serializing_if = "Vec::is_empty"`. the order doesn't matter, really, but a consistent one is a little nicer to read.

Reply:
Agree, fixed